js/form.js

```diff
@@ -0,0 +1,33 @@
+import {OFFER_PRICE} from './data.js'
+
+const apartmentTypeInput = document.querySelector('#type');
+const offerPriceInput = document.querySelector('#price');
+const checkinTimeInput = document.querySelector('#timein');
+const checkoutTimeInput = document.querySelector('#timeout');
```

Comment:
Элемент `form` содержит в себе ссылки на все инпуты. Если инпуты именованы, то хорошим решением было бы вынести форму в переменную и обращаться к ее инпутам через их `name`:
```suggestion
const formNode = document.querySelector('.ad-form');
// formNode.type
// formNode.timein
```
Так мы улучшим читаемость и производительность кода и не будем создавать лишних переменных